src/templates/injected.ts.tmpl

```diff
@@ -1,10 +1,4 @@
-import {createDeferredPromise} from '../util/DeferredPromise.js';
-
-declare global {
-  const InjectedUtil: {
-    createDeferredPromise: typeof createDeferredPromise;
-  };
-}
-
-/** @internal */
+/** 
+ * @internal
+ */
 export const source = SOURCE_CODE;
```

Comment:
Let's add a comment that the source code is produced by rollup and in which module format (CJS)?

Reply:
Done.